CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
 cmake_minimum_required (VERSION 3.10)
+
+#MSVC ABI (CMake 3.15+)
```

Comment:
```suggestion
# MSVC ABI (CMake 3.15+)
```
can you also change the other comments to have a leading space?

Also, we do have a minimum requirement of `3.10`, mostly because of android and old linux though. I think its fine to require a higher version on windows. Is it possible to make the `cmake_minimum_required` above conditional?

Reply:
changed